src/main/java/com/example/KitchenProject/Recipe.java

```diff
@@ -0,0 +1,20 @@
+package com.example.KitchenProject;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@AllArgsConstructor
+@Getter
+@Setter
+public class Recipe {
+    private String name;
+    private List<Ingredient> ingredients;
+    private String source;
+    private int time;
```

Comment:
What is the unit here? Minutes? If so, it would be better to name it `timeInMinutes` or maybe even `preparationTimeInMinutes`. I wonder if it will be easy to determine this prop for every recipe we will be adding to the database.